integration/spark/integrations/container/pysparkWriteIcebergTableVersionEnd.json

```diff
@@ -0,0 +1,29 @@
+{
+  "eventType" : "COMPLETE",
+  "job" : {
+    "namespace" : "testWriteIcebergTableVersion",
+    "name" : "open_lineage_integration_iceberg.append_data"
+  },
+  "outputs" : [ {
+    "namespace" : "file",
+    "name" : "/tmp/write_iceberg_table_version/db.table",
+    "facets" : {
+      "schema" : {
+        "fields" : [ {
+          "name" : "a",
+          "type" : "integer"
+        }, {
+          "name" : "b",
+          "type" : "integer"
+        } ]
+      },
+      "tableProvider" : {
+        "provider" : "iceberg",
+        "format" : "parquet"
+      }
+    },
+    "outputFacets" : {
+      "datasetVersion" : {}
```

Comment:
Iceberg's versions are apparently random longs - just checking if we have the facet here, since I can't deterministically tell which one it will be.